wiki/numerical-range/generalizations/c-perturbation-of-unitary-matrix-numerical-range.md

```diff
@@ -2,142 +2,158 @@
-changed depending on parameter $t$. Let $H := -i  ( U^V )$. Let us see
-that$H  (^d)$and$W(H) (-, \]$ for any $U,V \in
-\mathrm{U}(\mathbb{C}^d)$. We can also observe that  where
-\(\widetilde{U} := V^\dagger U V \in \mathrm{U}(\mathbb{C}^d)\).
-
-Hence, without loss of generality we can assume that $H$ is a diagonal
-matrix. Moreover, we can assume that $D \ge 0$ which follows from simple
-calculations  Let us see that the numerical range of $U(t)$ for any $t
-\in \[0,1\]$ is invariant to above calculations although the trajectory
-of $U(t)$ is changed. Therefore, we will consider the curve  where $t
-\in \[0,1\]$ and $U  (^d)$,$D\_+  (^d)$such that$D\_+ 0$. We will focus
-on the behavior of the spectrum of the unitary matrices $U(t)$, which
-will reveal the behavior of $W(U(t))$ for relatively small parameter
-$t$. Without loss of generality we can assume that $ ( D\_+ )=1$.
-Together with the fact that$D\_+  (^d)$and$D\_+ 0$ we can note that $D_+
-= \sum_{i=1}^{d} p_i \ket{i}\bra{i}$, where $p  ^d$ is a probability
-vector. Let us also define the set  for some matrix $M \in
-\mathrm{L}(\mathbb{C}^d)$ which consists of unit eigenvectors
-corresponding to the eigenvalue $\lambda$ of the matrix $M$. We denote
-by $k=r(\lambda)$ the multiplicity of eigenvalue $\lambda$ whereas by
-$I\_{M,} U(<sup>k,</sup>d)$ we denote the isometry which columns are
-formed by eigenvectors corresponding to eigenvalue $\lambda$ of a such
-matrix $M$. Let $(t), (t)  $for$t 0$. We will write$(t)  (t)$for
-relatively small$t 0$, whenever $\lambda(0)=\beta(0)$ and $(0)=(0)$.
-
-## Theorem
+# Perturbation of the numerical range of unitary matrices
 
-Let $U \in \mathrm{U}(\mathbb{C}^d)$ be a unitary matrix with spectral
-decomposition  Assume that the eigenvalue $\lambda \in \lambda(U)$ is
-such that $r(\lambda) = k$. Let us define a matrix $V(t)$ given by  Let
-$\lambda(t):=\lambda(UV(t))$ and let every $\_j(t)  (t)$corresponds to
-eigenvector$ $. Assume that $\lambda_{1}(t), \ldots, \lambda_{k}(t)$ are
-such eigenvalues that $\lambda_{j}(t) \to \lambda$, as $t \to 0$. Then:
+Let us consider two unitary matrices - matrix $U \in \mathrm{U}(\mathbb{C}^d)$
+and its perturbation $V \in \mathrm{U}(\mathbb{C}^d)$, such that $\\\|U-V\\\| \le \epsilon$, for some $\epsilon > 0$.
+To connect those unitary matrices, we define geodesic curve $U(t) \in \mathrm{U}(\mathbb{C}^d)$
+for any $t \in \[0,1\]$ with boundary conditions $U(0) \coloneqq U$ and $U(1) \coloneqq V$. 
+This smooth path is explicitly given as<sup>{% cite antezana2014optimal %}</sup> 
+
+\begin{equation}
+t \mapsto U \exp \left( t \text{Log} \left( U^\dagger V \right) \right). 
+\end{equation}
 
-1\. If $*{ S*^U} \_{i=1}^d\\ p\_i ||^2=0$, then $\lambda$ is an
-eigenvalue of $UV(t)$;
+We will study how the numerical range $W(U(t))$ changes depending on $t$. As shown in 
+{% cite kukulski2020 %} we may focus on equivalent, but simplified problem of determining the behavior
```

Comment:
Here the citation is on the normal level. Please change all citations to this style